plugins/destination/plugin.go

```diff
@@ -254,7 +254,13 @@ func (p *Plugin) Write(ctx context.Context, tables schema.Tables, sourceName str
 		panic("unknown client type")
 	}
 	if p.spec.WriteMode == specs.WriteModeOverwriteDeleteStale {
-		if err := p.DeleteStale(ctx, tables, sourceName, syncTime); err != nil {
+		include := func(t *schema.Table) bool { return true }
+		exclude := func(t *schema.Table) bool { return t.IsIncremental }
+		nonIncrementalTables, err := tables.FilterDfsFunc(include, exclude)
```

Comment:
Are `tables` already filtered by the glob patterns in the spec?
Making sure we don't return all tables here since we include everything.

Also since `include` is always `true`, maybe add only the exclude part for now?

Reply:
Right, `tables` are already filtered by glob patterns at this point (previously we were passing all these tables in to delete stale, now we're filtering out the incremental ones from that list first). 

`FilterDfsFunc` is now also used by `FilterDfs` internally, and that needs both `include` and `exclude`.